extension/content.css

```diff
@@ -586,6 +586,10 @@ div.inline-comment-form .form-actions,
 	float: none;
 }
 
+/* Add "copy filepath" button for PR page */
+.copy-filepath-btn:hover{
+	cursor: pointer;
+}
```

Comment:
@AshikNesin I meant a line between the two blocks so they're separated. This CSS isn't related to the "Add a Project" button CSS